beacon_node/src/cli.rs

```diff
@@ -250,7 +244,7 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .long("graffiti")
                 .help("Specify your custom graffiti to be included in blocks.")
                 .value_name("GRAFFITI")
-                .default_value(DEFAULT_GRAFFITI)
+                .default_value(lighthouse_version::VERSION)
```

Comment:
If we are going to do this, we should probably build a data type that ensures this is < `GRAFFITI_BYTES_LEN` so that it doesn't prevent clients from starting.

Reply:
It only prevents it from starting if the user-supplied string is too long. If our in-built string is too long it truncates.

Reply:
I think it errors if it's too long and the user doesn't specify

Reply:
Yeah you were right, I fixed it.